pkgs/development/python-modules/intelhex/default.nix

```diff
@@ -13,13 +13,6 @@ buildPythonPackage rec {
     sha256 = "0k5l1mn3gv1vb0jd24ygxksx8xqr57y1ivgyj37jsrwpzrp167kw";
   };
 
-  # FAIL: test_xrange_longint (intelhex.test.TestXrangeLongInt)
-  # AssertionError: OverflowError not raised
-  #
-  # Harmless test failure:
-  #  https://github.com/bialix/intelhex/issues/14
-  doCheck = false;
-
```

Comment:
It's my understanding that the test will only fail on 64-bit systems.  Please keep that in mind.